src/platform.deno.ts

```diff
@@ -116,6 +122,9 @@ async function* fetchFile(url: string | URL): AsyncIterable<Uint8Array> {
     }
     yield* body;
 }
+function isDenoFile(data: unknown): data is Deno.File {
+    return isDeno && data instanceof Deno.File;
```

Comment:
This is file is Deno-specific, why the runtime check?
```suggestion
    return data instanceof Deno.File;
```

Reply:
This file is web-specific. This does coincide with Deno (hence the name), but it will also be bundled for browsers.